.github/workflows/go.yml

```diff
@@ -0,0 +1,90 @@
+name: Go
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
```

Comment:
probably not needed

Reply:
I agree for Go code jobs.

Reply:
I've removed from pushes, but go checks should be on all PRs for all branches.